CONTRIBUTING.md

```diff
@@ -1,135 +1,107 @@
   
    
-   Remember, there's more to software development than code; if it's not properly planned, stuff gets messy real fast. 
+   Remember, there's more to software development than code; if it's not properly planned, stuff gets messy real fast.
 
 2. Create a fork and open a feature branch - `feature/my-cool-feature` is the classic way to name these, but it really doesn't matter.
 
 3. Create a pull request!
 
-4. Sign the [Contributor License Agreement](https://cla-assistant.io/loadimpact/k6) (the process is integrated with the pull request flow through cla-assistant.io)
+4. Sign the [Contributor License Agreement](https://cla-assistant.io/grafana/k6) (the process is integrated with the pull request flow through cla-assistant.io).
 
 5. We will discuss implementation details until everyone is happy, then a maintainer will merge it.
 
-Development setup
------------------
+### Development setup
 
-To get a basic development environment for Go and k6 up and running, first make sure you have **[Git](https://git-scm.com/downloads)** and **[Go](https://golang.org/doc/install)** (1.10 or newer) installed and working properly.
+To get a basic development environment for Go and k6 up and running, first make sure you have **[Git](https://git-scm.com/downloads)** and **[Go](https://golang.org/doc/install)** (see our [go.mod](https://github.com/grafana/k6/blob/master/go.mod#L3) for minimum required version) installed and working properly.
 
-Once that's done, you can get the k6 source into your Go workspace (`$GOPATH/src`) by running:
-```bash
-go get go.k6.io/k6
-```
-This will also build a `k6` binary and put it in `$GOPATH/bin`.
+We recommend using the Git command-line interface to download the source code for the k6:
 
-**Building from source**:
+* Open a terminal and run `git clone https://github.com/grafana/k6.git`. This command downloads k6's sources to a new `k6` directory in your current directory.
+* Open the `k6` directory in your favorite code editor.
 
-Standing in the repo root (`$GOPATH/src/go.k6.io/k6`) you can build a k6 binary from source by running:
-```bash
-cd $GOPATH/src/go.k6.io/k6
-go build
-```
+For alternative ways of cloning the k6 repository, please refer to [GitHub's cloning a repository](https://docs.github.com/en/github/creating-cloning-and-archiving-repositories/cloning-a-repository) documentation.
 
-**Running the linter**:
+#### Running the linter
 
-We make use of the [golangci-lint](https://github.com/golangci/golangci-lint) tool to lint the code in CI. To run it locally, first install it:
-```bash
-go get -u github.com/golangci/golangci-lint/cmd/golangci-lint
-```
-then run:
-```
-golangci-lint run --out-format=tab --new-from-rev master ./...
-```
+We make use of the [golangci-lint](https://github.com/golangci/golangci-lint) tool to lint the code in CI. To run it locally, first [install it](https://golangci-lint.run/usage/install/#local-installation), then run:
```

Comment:
Given the fact that the version here is pretty important maybe we can specify to get it from the first line of `.golangci.yaml` ?

Reply:
Maybe even preface it that they can just use the docker version as I would expect most users will read top-to-bottom, while doing the instructions. 

Maybe this can be rewritten to show they can skip that if they just use docker. 

WDYT @javaducky

Reply:
Yes, that's for sure. Let's reference here the line with the version.

I'm starting to think that it would be nice to have a tiny container where we can keep pre-installed tools with the desired versions and use it for the running things like `golangci-lint run --fix` or re-generate easy son, protobuf and so-on.

What do you think?

Reply:
I think that most people will not need anything but `golangci-lint` most of the time. This means that if we put more stuff in that image it will just be bigger for no benefit, most of the time. 

I am fine with there being another one for `go generate ./...` calling, but let's do that in a separate PR. Also, this happens *fairly* rare, but I guess it's a good idea to have the exact versions somewhere.